analyzers/tests/SonarAnalyzer.UnitTest/CFG/Roslyn/BasicBlockTest.cs

```diff
@@ -149,7 +149,7 @@ int Pow(int num, int exponent)
             body.Kind.Should().Be(BasicBlockKind.Block);
             body.Operations.Should().HaveCount(1).And.Subject.Single().Should().BeAssignableTo<IExpressionStatementOperation>();
             body.BranchValue.Should().BeAssignableTo<ILiteralOperation>();
-            body.OperationsAndBranchValue.Should().OnlyContainInOrder(body.Operations[0], body.BranchValue);
+            body.OperationsAndBranchValue.Should().Equal(body.Operations[0], body.BranchValue);
```

Comment:
education: so if the order is swapped, this will fail?

Reply:
Yes. See also the examples in https://fluentassertions.com/collections/